openedx/core/djangoapps/programs/signals.py

```diff
@@ -47,5 +47,5 @@ class of the object instance that sent this signal
         status,
     )
     # import here, because signal is registered at startup, but items in tasks are not yet able to be loaded
-    from openedx.core.djangoapps.programs import tasks
-    tasks.award_program_certificates.delay(user.username)
+    from openedx.core.djangoapps.programs.tasks.v1.tasks import award_program_certificates
```

Comment:
I've found that a nice pattern is to give the directory containing your versioned tasks the same name you would give a queue holding the tasks. For example, instead of `openedx.core.djangoapps.programs.tasks.v1.tasks`, I might use something like `openedx.core.djangoapps.programs.certification.v1.tasks`. What do you think?